README.md

```diff
@@ -1 +1,47 @@
-# display-input-data
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML Forms
+
+> This project consists of building an HTML document that matches the appearance of mint.com’s signup page.
+
+![screenshot][screenshot_url]
+
+[screenshot_url]: https://github.com/jrai0792/display-input-data/blob/main/127.0.0.1_60302_(CC's%20Laptop).png?raw=true
+
+Additional description about the project and its features.
```

Comment:
- [ ] **[MANDATORY]** Nice work including a template for your readme file 👍 just be careful: this line is just a placeholder in the template, you should really add more description about your project and it's features, for example, you can put the bullet points that you wrote in your pull request